api/HTMLSourceElement.json

```diff
@@ -117,10 +117,10 @@
               "version_added": "12"
             },
             "firefox": {
-              "version_added": "3.5"
+              "version_added": "15"
```

Comment:
This and https://developer.mozilla.org/en-US/docs/Web/HTML/Element/source might warrant a note, because at different times the `media` attribute on `<source>` has been used for `<video>` and `<picture>`, and I believe the latter is the only thing left supported in any engine. But `<picture>` wasn't added in Firefox 15, if https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#Browser_compatibility is to be trusted.

It might be that for some time the IDL attribute was there and merely reflecting the content attribute, even though it didn't do anything. If so, then a `partial_implementation` range seems appropriate.

Probably split this out?